protocols/discord.py

```diff
@@ -16,12 +16,16 @@
 # You should have received a copy of the GNU General Public License along
 # with this program. If not, see <http://www.gnu.org/licenses/>.
 import calendar
+from datetime import datetime
 import operator
 import collections
 import string
 import urllib.parse
 
 import socket, gevent.socket
+
+from pylinkirc.utils import split_hostmask
```

Comment:
I prefer `from pylinkirc import utils` (grouped with the other imports), just to prevent polluting the namespace. More likely it'd be sth like `from pylinkirc import structures, utils`